pkg/services/featuremgmt/registry.go

```diff
@@ -831,5 +831,12 @@ var (
 			Stage:        FeatureStageExperimental,
 			Owner:        grafanaBiSquad,
 		},
+		{
+			Name:         "kubernetesPlaylists",
+			Description:  "Enables use of kubernetes API for playlists",
```

Comment:
```suggestion
			Description:  "Use the kubernetes API in the frontend for playlists",
```

Current description sounds like it would control if the services is registered or not

Reply:
updated in 67c724b8eade9080e435d094fdf1b5b7198e6cbe